src/main/java/frc/robot/InputManager.java

```diff
@@ -41,7 +41,7 @@ public enum Button{
     }
 
     private static InputManager Instance;
-    private CommandGenericHID driveController;
+    public static CommandGenericHID driveController;
```

Comment:
please dont do this, the point of InputManager is to handle the controller.

Also, I dont see any use of it.

Reply:
Please remove these change is not needed.